### DISCUSSION TITLE
ordering the media list

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#294] [Ideas] [answered]
After collecting the media list and adding maybe a song or a custom video/picture to the list you have to order the media with the numbers.

It does work but it isn't quite souple to deal with.

Is it possible to click and drag the items in the media list in the right order before fetching the media?

Or, just an idea, maybe a good place to order media is after all media is downloaded and presented on the play screen. An advantage is that there you can see the thumbnails of the media too which makes it easier to identify media content.

Thanks for all your hard work Charles, many congregations here in the Netherlands are now using JWMMF together with OBS :-)

**Comment by sircharlo** [comment, marked as answer]
I like it. I'll see if if can add drag and drop reordering in the playback screen.
